docs/versioning/android-version.md

```diff
@@ -81,32 +81,36 @@ If true, omits zero in the patch version. So _42.10.0_ will become _42.10_ and _
+| Variable Name                  | Description                                                     |
+| ------------------------------ | --------------------------------------------------------------- |
+| `$AC_ANDROID_NEW_VERSION_CODE` | Represents the incremented version code applied to the project. |
+| `$AC_ANDROID_NEW_VERSION_NAME` | Represents the incremented version name applied to the project. |
 
 You can use the above values in the remaining steps of your workflow.
 
-### Best Practices
+### Input Variables
 
 The versioning system works by consuming environment variables. Even though it's easier to configure it by using UI, sometimes you may want to change them on the fly. Your commit messages or tags can be used to override those settings. The name of the variables and expected values can be found below.
 
-| Value                         | Explanation                                                                                                       |
-| ----------------------------- | ----------------------------------------------------------------------------------------------------------------- |
-| `$AC_BUILD_NUMBER_SOURCE`     | Version code source type(env,gradle)                                                                              |
-| `$AC_ANDROID_BUILD_NUMBER`    | Version code to set. If `$AC_BUILD_NUMBER_SOURCE` is set to gradle, this variable will be read from the project   |
-| `$AC_BUILD_OFFSET`            | The number to be added or subtracted from the `$AC_ANDROID_BUILD_NUMBER`                                          |
-| `$AC_VERSION_NUMBER_SOURCE`   | Version name source type(env,gradle)                                                                              |
-| `$AC_ANDROID_VERSION_NUMBER`  | Version name to set. If `$AC_VERSION_NUMBER_SOURCE` is set to gradle, this variable will be read from the project |
-| `$AC_VERSION_STRATEGY`        | Version Increment Strategy                                                                                        |
-| `$AC_VERSION_OFFSET`          | The number to be added or subtracted from the `$AC_ANDROID_VERSION_NUMBER`                                        |
-| `$AC_FLAVOR`                  | Build flavor                                                                                                      |
-| `$AC_OMIT_ZERO_PATCH_VERSION` | If true omits zero in patch version                                                                               |
+<Screenshot url='https://cdn.appcircle.io/docs/assets/android-workflow-components-increment-build-and-version-number_2.png' />
+
+| Variable Name                 | Description                                                                                                                                                                                                                                                                     | Status   |
```

Comment:
could you please check the required status for each parameter. In component repository, only repo dir has specified as Required.